source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -8307,6 +8309,7 @@ export class AdaptiveCard extends ContainerWithActions {
     onImageLoaded?: (image: Image) => void;
     onInlineCardExpanded?: (action: ShowCardAction, isExpanded: boolean) => void;
     onInputValueChanged?: (input: Input) => void;
+    onCarouselEvent?: (carouselEvent: CarouselEvent) => void;
```

Comment:
For telemetry, we may not need to implement it in AdaptiveCard level. `static` event is enough

Reply:
okay